### DISCUSSION TITLE
Unity3D Repaint issue when switching workspace

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#6059] [Q&A]
When I switch to and from my Unity workspace, the editor does not repaint properly. It only refreshes when I hover my mouse over the panel or toggle the application fullscreen and back.

I've seen various recommendations online, with many suggesting the use of a compositor. I am using Picom and have ensured it's rendering with GLX. I've also tried other backends, but the issue persists. I have also tried using the "reload" function in i3 when switching to the workspace, but it doesn't seem to do anything. 

Has anyone found a solution to this?

**Comment by orestisfl** [comment]
> I've seen various recommendations online, with many suggesting the use of a compositor. I am using Picom and have ensured it's rendering with GLX. I've also tried other backends, but the issue persists.

Did you try without any compositor at all?

**Reply by xoorath** [reply]
Sorry to necro this thread - I have the same experience as ghost with and without a compositor.

This thread reports the issue as well: https://discussions.unity.com/t/editor-panels-only-redraw-on-update-mouse-over/755222/21?page=2

From my reading of that thread it seems like this is an issue with Unity, not the WM or compositor.
